app/src/pages/inside/profilePage/personalInfoBlock/userInfo/userInfo.jsx

```diff
@@ -29,10 +29,13 @@ const messages = defineMessages({
 });
 
 @connect(
-  (state) => ({
-    name: userInfoSelector(state).full_name,
-    email: userInfoSelector(state).email,
-  }),
+  (state) => {
+    const userInfo = userInfoSelector(state);
+    return {
```

Comment:
Please make dedicated selectors for these fields

Reply:
fixed